addons/website/views/website_templates.xml

```diff
@@ -389,14 +389,107 @@
 </template>
 
 <template id="template_header_hamburger" inherit_id="website.layout" name="Template Header Hamburger" active="False">
-    <xpath expr="//header//nav[hasclass('navbar')]" position="attributes">
-        <attribute name="class" remove="navbar-expand-md" separator=" "/>
-    </xpath>
-    <xpath expr="//header//button[hasclass('navbar-toggler')]" position="replace">
-        <div class="o_navbar_toggler_container">$0</div>
-    </xpath>
-    <xpath expr="//header//ul[@id='top_menu']" position="attributes">
-        <attribute name="class" separator=" " remove="text-right"/>
+    <xpath expr="//header//nav" position="replace">
+        <nav class="navbar navbar-light o_colored_level o_cc o_cc2"> <!--  (╯°□°)╯︵ ┻━┻  -->
```

Comment:
Yes... don't use o_cc2, that color is done in scss

Reply:
No possible solution to define a default color/palette for an header template? 😢

Reply:
Same debate as for nav link style. You probably could with data-trigger and data-trigger-value (to indicate the color)... but is it really needed? Priority on finishing the task, this is a detail we can improve later

Reply:
Indeed, it's just a "nice to have" stuff not a priority.
Is it really needed? Yes, it is 😄